docs/commands/config.md

```diff
@@ -43,6 +43,7 @@ Syntax: `config {flags}`
 | rm_always_trash    | boolean                | whether or not to always use system trash when no flags are given to `rm` |
 | pivot_mode         | "auto" or "always" or "never"                | "auto" will only pivot single row tables if the output is greater than the terminal width. "always" will always pivot single row tables. "never" will never pivot single row tables.            |
 | plugin_dirs        | table of strings       | additional directories to search for plugins during startup               |
+| cal_week_start_day | string ("sunday" ...)  | Sets the day that the week starts on in the calendar                      |
```

Comment:
We've been pushing for less configuration and more flags in commands. It has some advantages:

- Help is closer to where people would expect it,
- Far easier to temporarily override,
- Defaults can be made with aliases, once https://github.com/nushell/nushell/issues/1885 is resolved.

What do you think about making this a flag instead?